squad/api/rest.py

```diff
@@ -571,6 +572,32 @@ def unsubscribe(self, request, pk=None):
         data = {"email": subscriber_email}
         return Response(data, status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=['get', 'post'], suffix='compare_builds')
+    def compare_builds(self, request, pk=None):
+        builds_to_compare = {
+            "baseline": request.data.get("baseline", None) if request.method == 'POST' else request.query_params.get("baseline", None),
+            "to_compare": request.data.get("to_compare", None) if request.method == 'POST' else request.query_params.get("to_compare", None)
+        }
+        if any(builds_to_compare.values()) and all(builds_to_compare.values()):
+            try:
+                int(builds_to_compare['baseline'])
+                int(builds_to_compare['to_compare'])
+                baseline = self.get_object().builds.get(pk=builds_to_compare['baseline'])
+                if not baseline.status.finished:
+                    raise serializers.ValidationError("Cannot report regressions/fixes on a non-finished baseline build")
+                to_compare = self.get_object().builds.get(pk=builds_to_compare['to_compare'])
```

Comment:
I think to_compare should also be finished.

Reply:
I agree with Milosz

Reply:
ah..was wondering too. thanks